packages/amplify-graphql-searchable-transformer/src/__tests__/__snapshots__/amplify-graphql-searchable-transformer.tests.ts.snap

```diff
@@ -97,12 +97,12 @@ input SearchablePostSortInput {
 
 type SearchablePostConnection {
   items: [Post]
-  nextToken: String
+  nextToken: [String]
```

Comment:
I think the plan was to keep `nextToken` as a string. I think we can use the VTL functions for encoding and decoding to do this transparently.

Reply:
Updated to base64 string.